examples/frontend/public/logo.svg

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
How did you export this?

Reply:
I didn't export it, it's served publicly by NextJs, I just added a URI to point to it.
https://nextjs.org/docs/basic-features/static-file-serving

Reply:
https://github.com/curaOS/source/blob/2766b8d9415b24aaef0547bd3965896bb1fe616a/examples/frontend/theme.ts#L2